bootstrap.sh

```diff
@@ -64,3 +64,23 @@ do
     && cp -n Local.templates/$CONFIG_FILE Local/$CONFIG_FILE \
   )
 done
+
+# Build Yubikit
+YUBIKIT_DIR=Carthage/Checkouts/yubikit-ios
+YUBIKIT_OUT=ThirdParty/YubiKit
+
+rm -rf YUBIKIT_DIR
```

Comment:
1. These lines are not evaluating to the value because it is not prefixed with `$`
2. These two lines will error if the directories don't exist. If we add `set -x` to the top of this file (which will happen eventually), it would stop here.

Directory should be checked that it exists before it is attempted to be removed:

```
[ -d "$YUBIKIT_DIR" ] && rm -rf "$YUBIKIT_DIR"
```

Reply:
> Sorry one last small thing. Also I dont see the change for making sure the directories are there before removing them?
> 
> ```shell
> [ -d "$YUBIKIT_DIR" ] && rm -rf "$YUBIKIT_DIR"
> [ -d "$YUBIKIT_OUT" ] && rm -rf "$YUBIKIT_OUT"
> ```

I don't think we need to check if the directory exists, `rm` will do nothing if the directory does not exist. Let me know if you have any concerns.

Reply:
Sorry, didn't realize that `-f` overrides this and does not fail if the directory doesn't exist. Should be good then